pkg/cmd/roachprod/main.go

```diff
@@ -885,6 +885,28 @@ var getProvidersCmd = &cobra.Command{
 	},
 }
 
+var startGrafanaCmd = &cobra.Command{
```

Comment:
When trying this out with local roachprod clusters on my macbook, things don't work. Short of trying to make a version of this work locally (probably too annoying for now), we should at least error out more informatively.